monitoring/alerts/spark-rules.yml

```diff
@@ -0,0 +1,98 @@
+groups:
+- name: Spark_Alerts
+  rules:
+    - alert: Spark_Dispatcher_Count
+      expr: count(spark_mesos_cluster_driver_launched) >= 50
+      for: 1m
+      labels:
+        severity: major
```

Comment:
why not informational?

Reply:
no reason other than I didn't know that was a better alternative. I'll change all the 'good' alerts to informational.